src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/PluginIT.java

```diff
@@ -0,0 +1,95 @@
+
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.client.ResponseException;
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.test.ESIntegTestCase;
+
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static com.amazon.opendistroforelasticsearch.sql.intgtest.TestsConstants.TEST_INDEX_ACCOUNT;
+import static com.amazon.opendistroforelasticsearch.sql.esintgtest.TestUtils.SettingType;
+import static org.hamcrest.Matchers.equalTo;
+
+public class PluginIT extends SQLIntegTestCase {
+
+    @Override
+    protected void init() throws Exception {
+        loadIndex(Index.ACCOUNT);
+    }
+
+    @Test
+    public void sqlEnableSettingsTest() throws IOException {
+        updateClusterSettings(SettingType.PERSISTENT, "opendistro.sql.enabled", "true");
+        JSONObject queryResult = executeQuery(String.format("SELECT firstname FROM %s/account WHERE account_number=1", TEST_INDEX_ACCOUNT));
+        System.out.println(queryResult);
+        Assert.assertThat(getHits(queryResult).length(), equalTo(1));
+
+        updateClusterSettings(SettingType.PERSISTENT, "opendistro.sql.enabled", "false");
+        Response response = null;
+        try {
+            queryResult = executeQuery(String.format("SELECT firstname FROM %s/account WHERE account_number = 1", TEST_INDEX_ACCOUNT));
+            System.out.println(queryResult);
+        } catch (ResponseException ex) {
+            response = ex.getResponse();
+        }
+
+        queryResult = new JSONObject(TestUtils.getResponseBody(response));
+        Assert.assertThat(queryResult.getInt("status"), equalTo(400));
+        JSONObject error = queryResult.getJSONObject("error");
+        Assert.assertThat(error.getString("reason"), equalTo("Invalid SQL query"));
+        Assert.assertThat(error.getString("details"), equalTo("Either opendistro.sql.enabled or rest.action.multi.allow_explicit_index setting is false"));
+        Assert.assertThat(error.getString("type"), equalTo("SQLFeatureDisabledException"));
+        resetClusterSettings(SettingType.PERSISTENT, "opendistro.sql.enabled");
+    }
+
+    private JSONObject updateClusterSettings(SettingType settingType, String setting , String value) throws IOException {
+        Request request = new Request("PUT", "/_cluster/settings?pretty");
+        String persistentSetting = "{\""+ settingType.val() +"\": {\"" + setting + "\": " + value + "}}";
+        request.setJsonEntity(persistentSetting);
+        RequestOptions.Builder restOptionsBuilder = RequestOptions.DEFAULT.toBuilder();
+        restOptionsBuilder.addHeader("Content-Type", "application/json");
+        request.setOptions(restOptionsBuilder);
+        JSONObject result = new JSONObject(executeRequest(request));
+        System.out.println(result);
```

Comment:
remove

Reply:
This is needed for debugging. We do have  `System.out.println();` in other parts of the test codebase as well.

Reply:
We shouldn't. We can add it locally when needed, but we shouldn't check in. We could have a logger configured for tests that will output to console, but I think it would be more useful if it weren't easy to change the code while debugging, whereas in this case you can always update the code and run locally, so I don't think there's a good reason to keep any of the `System.out.print*` calls.

Reply:
Will remove.